src/test/java/seedu/address/model/person/SocialHandleTest.java

```diff
@@ -13,19 +13,12 @@ public void constructor_null_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () -> new SocialHandle(null));
     }
 
-    @Test
-    public void constructor_invalidSocialHandle_throwsIllegalArgumentException() {
-        String invalidSocialHandle = "";
```

Comment:
Would it be better to change the invalidSocialHandle to something else instead of removing this test?